gui/default/syncthing/settings/settingsModalView.html

```diff
@@ -70,15 +70,15 @@
             <div class="col-md-6">
               <div class="form-group">
                 <label translate for="urVersion">Anonymous Usage Reporting</label> (<a href="" translate data-toggle="modal" data-target="#urPreview">Preview</a>)
-                <div ng-if="tmpOptions.upgrades != 'candidate'">
+                <div ng-if="tmpOptions.upgrades != 'candidate' && !version.isCandidate">
```

Comment:
Whats the point of the upgrades check in this case?

Reply:
This shows the "usage reporting always on" variant (no choice, and corresponding help text) when upgrades to candidates is selected. That's not a change.